File: loops/fizzbuzz.py
# Malu Estevam, fizzbuzz Python
for x in range(1,51):
    if x % 3:
        print("fizz")
    if x % 5:
        print("buzz")
    if x % 3 and x % 5:
        print("fizzbuzz")
    else:
        print(x)
    

File: loops/fizzbuzz.c
//Malu Estevam, FizzBuzz C
#include <stdio.h>

int x;

int main(void){
    for(x=0;x<51;x++){
        if (x % 3 == 0 && x % 5 == 0){
            printf("fizzbuzz\n");

        }
        else if (x % 3 == 0){
            printf("fizz\n");

        }
        else if (x % 5 ==0){
            printf("buzz\n");

        }else{
            printf("%d\n", x);
        }
    }
    return 0;
}